### PR TITLE
I108 improve website accessibility score

### LINKED ISSUE #108: Improve website accessibility score
## Basic Information
From looking at lighthouse in the browser devtools our accessibility score is not great
We need to add aria and role attributes to elements
https://vuetifyjs.com/en/features/accessibility/

Screen Shot 2023-07-10 at 1 31 00 pm

### PR DESCRIPTION
**Pull request by makon1234**
## Change Summary
Improved score by mostly adding aria roles to specific elements

### Change Form
NA

- [x ] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
Background and foreground colors do not have a sufficient contrast ratio is the main problem. 

# Related Issue

- Resolve #108

### CODE REVIEW

File: client/src/components/ChangePasswordModal.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <v-dialog width="500px">
     <v-card class="bg-backgroundGrey">
-      <v-img src="/images/Footer-min.jpeg" width="100%" max-height="16" cover />
+      <v-img src="/images/Footer-min.jpeg" width="100%" max-height="16" alt="" cover/>
```
Comment (ChantelleKerr) [inline client/src/components/ChangePasswordModal.vue:4]: I would still give this a name, theres a few other places where the alt is empty

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Looks good so far. Some alt attributes are empty. You will need to run prettier to lint, `yarn lint:format` in client dir

File: client/src/components/ChangePasswordModal.vue (hunk 1 above)
Comment (makon1234) [reply client/src/components/ChangePasswordModal.vue]: image

Comment (makon1234) [reply client/src/components/ChangePasswordModal.vue]: I am trying to fix this error, do you know how to?

Comment (makon1234) [reply client/src/components/ChangePasswordModal.vue]: image
Package.json already seems to have it

**Comment by makon1234** [comment]
never mind

**Review by ChantelleKerr — APPROVED** [review]
Good job. Approved

**Pull request merged**